src/__tests__/downshift.get-item-props.js

```diff
@@ -161,32 +167,74 @@ test(`getItemProps doesn't include event handlers when disabled is passed (for I
   }
 })
 
+test(`disabled item can't be selected by pressing enter`, () => {
+  const items = [
+    {item: 'Chess', disabled: true},
+    {item: 'Dominion', disabled: true},
+    {item: 'Checkers', disabled: true},
+  ]
+  const utils = renderDownshift({items})
+  const {input, arrowDownInput, enterOnInput, changeInputValue} = utils
+
+  const firstItem = utils.queryByTestId('item-0')
+  expect(firstItem.hasAttribute('disabled')).toBe(true)
+
+  // input.simulate('keydown')
```

Comment:
This comment here doesn't add anything I think. The name of the function is sufficient documentation :)

Reply:
👍